### PR TITLE
Add Object.linkingObjectCount()

### PR DESCRIPTION
**Pull request by astigsen**
## What, How & Why?

This adds `Object.linkingObjectCount()` method to get the total number of incoming links to an object.

## ☑️ ToDos

* [x] 📝 Changelog entry
* [x] 📝 `Compatibility` label is updated or copied from previous entry
* [x] 🚦 Tests
* [x] 📝 Public documentation PR created or is not necessary
* [x] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [x] typescript definitions file is updated
* [x] jsdoc files updated
* [x] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

**Comment by astigsen** [comment]
I also thought of naming it `backlinksCount()`, but in the end I found it more correct to align the naming with the other method on the object which is called `linkingObjects()` (in .NET this looks to be named `GetBacklinks()`, so there it would match up).

File: docs/object.js [hunk 1]
```diff
@@ -46,4 +46,11 @@ class Object {
      * @since 1.9.0
      */
     linkingObjects(objectType, property) {}
+
+    /**
+     * Returns the total count of incoming links to this object
+     * @returns {number} number of links to this object.
+     * @since 2.6.0
+     */
+    linkingObjectsCount(objectType, property) {}
```
Comment (kneth) [inline docs/object.js]: The parameter should be documented too. They can be copied from the method above.

**Comment by kneth** [comment]
One - unrelated - test is failing in one build. I have seen it before, and I haven't a clear understanding of why it fails. I believe we can ignore it for now (see also #1748).

File: docs/object.js (hunk 1 above)
Comment (astigsen) [reply docs/object.js]: Yes, it should not take any parameters. It was a copy/paste mistake. Fixed.

**Pull request merged**